src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+      class="tw-mr-12"
+    >
+      <ec-icon
+        class="ec-summary-info__main-icon"
+        :data-test="`ec-summary-info__main-icon ec-summary-info__main-icon__${iconName}`"
+        :name="iconName"
+        :size="16"
+      />
+    </div>
+
+    <slot>
+      <div
+        v-if="lineItems.length > 0"
+        class="ec-summary-info__content"
+      >
+        <div
+          data-test="ec-summary-info__content-lines"
+        >
+          <div
+            v-for="(item, index) in lineItems"
+            :key="item.text"
+            class="ec-summary-info__content-line-item"
+          >
+            <span
+              :title="item.text"
+              class="ec-summary-info__content-line-item-content"
+              :data-test="`ec-summary-info__content-line-item-content ec-summary-info__content-line-item-content-${getStylePreset(item.stylePreset, index)}`"
+              :class="`ec-summary-info__content-line-item-content-${getStylePreset(item.stylePreset, index)}`"
+            >
+              {{ item.text }}
+            </span>
+
+            <span
```

Comment:
Hi, can we please double check we need this span?

Probably we can go away with just 

```
            <ec-icon
              v-if="item.tooltipText"
              v-ec-tooltip="{
                content: item.tooltipText
              }"
              :class="`ec-summary-info__content-line-item-icon-${getStylePreset(item.stylePreset, index)}`"
              :data-test="`ec-summary-info__content-line-item-icon ec-summary-info__content-line-item-icon-${getStylePreset(item.stylePreset, index)}`"
              class="ec-summary-info__content-line-item-icon tw-ml-4"
              name="simple-info"
              :size="14"
            />
```

Reply:
ok corrected, thanks